### ISSUE TITLE
`zcash_client_backend`: Support import of seed phrases from transparent-only wallets. 

### ISSUE THREAD

**Issue by nuttycom** [zcash/librustzcash#1188]
> More specifically, we realized as part of the discussion that there is probably an existing bug related to the import of seeds from transparent-only wallets, where we are not at present scanning any of the internal key tree (which is where Bitcoin wallets regularly send change) so if you import a seed that was previously used with a transparent-only wallet, you're likely to be missing funds.

> Then again, if you do that, then you're also not doing gap limit scanning at the moment, so _both_ of these should be addressed at once. I'll open an issue for that and it can get stuck in the backlog somewhere.

_Originally posted by @nuttycom in https://github.com/zcash/librustzcash/pull/1185#discussion_r1488643222_
